src/main/java/online/book/store/dto/CreateBookRequestDto.java

```diff
@@ -1,15 +1,26 @@
 package online.book.store.dto;
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
+import jakarta.validation.constraints.Size;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class CreateBookRequestDto {
-    private String title;
+    @NotNull
+    @Size(min = 5, message = "author size must be min 5")
```

Comment:
You can also add the maximum number of characters, which is set in the field property in the database. If I'm not mistaken default is 255 characters